src/ray/core_worker/transport/concurrency_group_manager.h

```diff
@@ -35,7 +35,8 @@ class ConcurrencyGroupManager final {
  public:
   explicit ConcurrencyGroupManager(
       const std::vector<ConcurrencyGroup> &concurrency_groups = {},
-      const int32_t max_concurrency_for_default_concurrency_group = 1);
+      const int32_t max_concurrency_for_default_concurrency_group = 1,
+      std::function<std::function<void()>()> initializer = nullptr);
```

Comment:
please keep the same name through the whole call stack
```suggestion
      std::function<std::function<void()>()> initialize_thread_callback = nullptr);
```

Reply:
oh sorry, I am still polishing the draft PR.